spot_utils/controllers/startup.py

```diff
@@ -0,0 +1,54 @@
+"""Functions for starting up and controlling the lease for spot."""
+
+import bosdyn.client
+import bosdyn.client.estop
+import bosdyn.client.lease
+import bosdyn.client.util
+from bosdyn.client.image import ImageClient
+from bosdyn.client.manipulation_api_client import ManipulationApiClient
+from bosdyn.client.robot_command import RobotCommandClient
+from bosdyn.client.robot_state import RobotStateClient
+
+from spot_utils.structures.robot import RobotClient
+
+LIS_SPOT_IP = "192.168.80.3"
```

Comment:
in case anyone has the questions I asked about this on slack, maybe we can rename this to `DEFAULT_SPOT_IP`?